website/docs/administration/context/index.mdx

```diff
@@ -59,4 +59,26 @@ Once connected, the indexing job will start automatically. You can check the sta
 
 Additionally, you can also visit the **Code Browser** page to view the connected repository.
 
-<img src={CodeBrowserUrl} alt="code browser" width="800" />
\ No newline at end of file
+<img src={CodeBrowserUrl} alt="code browser" width="800" />'
+
+# Embedding
+
+Embedding is used to convert your search queries or chat messages into a vector of numbers that can be used to find relevant context.
+
+## Adding an embedding model provider
+
+You can add a new embedding model provider by adding a new section to the `~/.tabby/config.toml` file.
```

Comment:
please also briefly introduce how embedding is used in indexing: https://github.com/TabbyML/tabby/blob/1cf971c0bd739f1a6c265198ecf71202a9941896/crates/tabby-scheduler/src/doc/mod.rs#L86